lib/Backends/OpenCL/OpenCL.h

```diff
@@ -17,6 +17,7 @@
 #define GLOW_BACKENDS_OPENCL_OPENCL_H
 
 #include "glow/Backends/Backend.h"
+#include "glow/Backends/BackendUtils.h"
```

Comment:
Same as CPU -- can this be moved to `OpenCL.cpp` instead of including in the header?